api/app/views/mno_enterprise/jpi/v1/impac/dashboards/_dashboard.json.jbuilder

```diff
@@ -1,4 +1,6 @@
-json.extract! dashboard, :id, :name, :full_name, :currency
+json.extract! dashboard, :id, :full_name, :currency
```

Comment:
I think this is a mistake: I don't see why the dashboard name should be removed while it's still has to be editable.
Probably there has been a confusion with `kpi.name` which has to be removed.

I'll revert this line